open-api/rest-catalog-open-api.yaml

```diff
@@ -120,6 +122,32 @@ paths:
         5XX:
           $ref: '#/components/responses/ServerErrorResponse'
 
+  /v1/oauth/token:
+
+    post:
+      tags:
+        - Auth API
+      summary: Get a token using an OAuth2 flow
+      operationId: getToken
+      description:
+        Exchange credentials for a token using the OAuth2 client credentials flow or token exchange.
+      requestBody:
+        content:
+          application/json:
```

Comment:
This should be `application/x-www-form-urlencoded`

Reply:
Fixed!